llvm/test/Analysis/ValueTracking/knownbits-trunc-with-min-max-clamp.ll

```diff
@@ -0,0 +1,174 @@
+; RUN: opt < %s -passes=aggressive-instcombine -mtriple=x86_64 -S | FileCheck %s
+
+; This LIT test checks if TruncInstCombine pass correctly recognizes the
+; constraints from a signed min-max clamp. The clamp is a sequence of smin and
+; smax instructions limiting a variable into a range, smin <= x <= smax.
+
+declare i16 @llvm.smin.i16(i16, i16)
```

Comment:
Declaration of intrinsics is unnecessary.

Reply:
Removed.